R/centralization.R

```diff
@@ -397,8 +402,8 @@ centr_degree <- function(
 #' @examples
 #' # A BA graph is quite centralized
 #' g <- sample_pa(1000, m = 4)
-#' centr_degree(g, normalized = FALSE)$centralization %>%
-#'   `/`(centr_degree_tmax(g, loops = FALSE))
+#' centr_degree(g, normalized = FALSE)$centralization /
+#'   centr_degree_tmax(g, loops = "twice")
```

Comment:
I remember I wasn't sure about this. For `centr_degree()`, the default is `"twice"` IIRC. Is then the same `loops` value needed in the denominator?

Reply:
Yes, this is correct.